util/bisect

```diff
@@ -1,6 +1,16 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+seed = begin
+         Integer(ENV["SEED"])
+       rescue
+         abort "Specify the failing seed as the SEED environment variable"
+       end
+
+gemdir = `gem env gemdir`.chomp
```

Comment:
Does `gemdir = Gem.dir` do the same?

Reply:
Absolutely, not sure why I did that in the first place. Will update, thanks! :+1: